src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -215,6 +216,12 @@ int ExecuteManagedAssembly(
     coreClrDllPath.append("/");
     coreClrDllPath.append(coreClrDll);
 
+#if defined(__LINUX__)
+    std::string coreClrTPproviderDllPath(clrFilesAbsolutePath);
```

Comment:
The messaging for this should be to update the WIKI documentation, about enabling tracing, to callout how hosts need to be updated to make similar change if they wish to support tracing scenarios.

@davidfowl - FYI - as this will impact DNX host.